### PR TITLE
Feat: vsdspub-560: kafka in

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

**Comment by Yalz** [comment]
Testing will come from joined Kakfa in/out ITs (to be created after this merge)

File: ldi-orchestrator/ldio-connectors/ldio-kakfa-in/pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <properties>
+        <relative-formatter-path>../../..</relative-formatter-path>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+            <version>3.0.5</version>
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-connectors/ldio-kakfa-in/pom.xml:22]: can you use a property for this?

File: ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioKafkaIn.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiInput;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.listener.MessageListener;
+
+public class LdioKafkaIn extends LdiInput implements MessageListener<String, String> {
+	private final String contentType;
+
+	/**
+	 * Creates a LdiInput with its Component Executor and LDI Adapter
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioKafkaIn.java:13]: This is quite generic, maybe say it is the kafka impl of LdiInput?

File: ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaInAutoConfig.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldio.LdioKafkaIn")
+	public LdioKafkaInConfigurator ldioConfigurator() {
+		return new LdioKafkaInConfigurator();
+	}
+
+	public static class LdioKafkaInConfigurator implements LdioInputConfigurator {
+
+		@Autowired
+		ConfigurableApplicationContext configContext;
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaInAutoConfig.java:37]: what is this used for?

**Review by Tomvbe — APPROVED** [review]

**Pull request merged**